command/repo.go

```diff
@@ -87,6 +89,41 @@ With no argument, the repository for the current directory is displayed.`,
 	RunE: repoView,
 }
 
+func parseExtraArgs(extraArgs []string) (args []string, target string) {
```

Comment:
Nit: this function has a generic-sounding name, but applies specifically to parsing clone arguments (it makes little sense otherwise). Since this function is shared across all `command` package, which is our biggest package, we could benefit from it having a more specific name, such as `parseCloneArgs`

Reply:
Ah right, `go` newb mistake ;) Fixed in 6428e3da10d6a5c59d252033d14da34457aa5d53